docs/user-guide/agile-boards/README.md

```diff
@@ -39,22 +41,44 @@ The new Boards are tightly integrated with all other project management function
 
 ## Create a new Board
 
-You can create as many flexible boards in a project as you need and configure them to your needs. First, you need to create a new Boards view. 
-If you haven't done so yet, [activate the Boards module](../projects/project-settings/modules) within your project. Also, we recommend to verify [Roles and Permissions](https://www.openproject.org/help/administration/manage-roles-permissions/) within your system's Administration.
+You can create as many Agile Boards in a project as you need and configure them to your needs. First, you need to create a new Boards view. 
+If you haven't done so yet, [activate the Boards module](../projects/project-settings/modules) within your project. Also, we recommend to verify [Roles and Permissions](../../system-admin-guide/users-permissions/roles-permissions/) within your system's Administration.
 
 Click on the green **+Board** button to create a new Board view. 
 
 ![OpenProject-Boards_create-new](OpenProject-Boards_create-new.png)
 
-## Choose between Basic board and Action board
+## Choose between board types
+
+Next, you need to choose which kind of Agile Board you want to create.
+
+#### Basic board
+You can freely create lists, name them and order your work packages within. If you move work packages between  the lists, there will be **NO changes** to the work package itself. This allows you to create flexible boards for any kind of activity you would like to track, e.g. Management of Ideas.
+
+#### Action boards
+
+In an Action board each lists represents a value of an attribute of the contained work packages (cards), e.g. there's a list for the status "New" and a list for the status "In Progress" in the Status board.
+Moving work packages (cards) between two lists will update them automatically, based on the list to which they're moved.
+After [adding lists to your board](#add-lists-to-your-board) they will automatically be filled with the respective work packages.
+
+There are several **types of Action boards** available:
+
+**Status board:** Each list represents a status. That means that e.g. all work packages with the status new will be automatically displayed in the column "New". 
```

Comment:
I would add quotation marks around status "new" to make it a bit clearer what is meant.